### DISCUSSION TITLE
Can you install SO 2 on ESXi

### DISCUSSION THREAD

**Discussion by Jcremer11** [Security-Onion-Solutions/securityonion#1835] [Unsupported Versions] [answered]
We have a relatively beafy server / host. I wold like to install SO 2 as 2 separate VMs...1 Management VM and 1 Heavy Node VM. 

Has anyone tried this? Is it even possible to do without having a bunch of performance issues? 

Basically - a fully distributed deployment is not going to be possible for us right now...especially if it requires 3 separate server boxes to make it happen. 

Any assistance would be greatly appreciated. 
Thanks...and Keep up the great work :)

**Comment by TOoSmOotH** [comment, marked as answer]
You can use VMs on all the components on a single physical machine. I would recommend passing the PCI bus for the monitoring NIC through to the VM.
